server/config/mongoid.yml

```diff
@@ -25,6 +25,10 @@ production:
         max_pool_size: 40
         write:
           w: 1
+        ssl_verify: <%= ENV['MONGODB_SSL_VERIFY'] || true %>
+        <% if ENV['MONGODB_SSL_CA_CERT'] %>
```

Comment:
Maybe use `-%>` to avoid creating blank lines? Same for `end`, although it shouldn't matter in YAML

Reply:
I _think_  mongoid run the template through only in memory, at least I don't see it being modified on the container:
```
docker run -ti --rm server_api cat config/mongoid.yml
development:
  clients:
    default:
      uri: <%= ENV['MONGODB_URI'] || 'mongodb://localhost:27017/kontena_development' %>
      options:
        max_pool_size: 25
  options:
    use_utc: true
    raise_not_found_error: false
test:
  clients:
    default:
      uri: <%= ENV['MONGODB_URI'] || 'mongodb://localhost:27017/kontena_test' %>
      options:
        max_pool_size: 25
        write:
          w: 1
  options:
    use_utc: true
    raise_not_found_error: false
production:
  clients:
    default:
      options:
        max_pool_size: 40
        write:
          w: 1
        ssl_verify: <%= ENV['MONGODB_SSL_VERIFY'] || true %>
        <% if ENV['MONGODB_SSL_CA_CERT'] %>
        ssl_ca_cert: <%= ENV['MONGODB_SSL_CA_CERT'] %>
        <% end %>
      uri: <%= ENV['MONGODB_URI'] || ENV['MONGOHQ_URL'] %>
  options:
    use_utc: true
    raise_not_found_error: false

```

Reply:
https://github.com/mongodb/mongoid/blob/master/lib/mongoid/config/environment.rb#L38-L40

Reply:
so the extra line breaks should not really matter as no one sees them :D